src/Page.js

```diff
@@ -150,7 +131,7 @@ function formatSiteNav(renderedSiteNav, src) {
     return renderedSiteNav;
   }
   // Tidy up the style of the unordered list <ul>
-  listItems.parent().attr('style', 'list-style-type: none; margin-left:-1em');
+  listItems.parent().addClass('site-nav-list');
```

Comment:
Should we move `site-nav-list` to a constant? (similar to `page-nav`)

Reply:
Actually this brings up an interesting point.

Relating to Webstorm IDE (maybe other IDEs as well), where if you refactor CSS names, it will automatically change the name of that class within html strings or function parameters that take in CSS names. However, constant strings with the same name will not be refactored. 

Also, we shouldn't use a constant for a string if it is only used once throughout the whole file. 

So we should be finding single use constants and changing them back to be used inline? (`site-nav-list` only appears once)

Reply:
Hmm, I would say that there's still value in refactoring it even if it is a single use now, because in case in the future we need to use this somewhere else in the code, we need not do the refactoring again (which is also provided that we even realise that the same class is used here. If we forgot about it, then we are using "magic values" everywhere).